### PR TITLE
Quad status

### PR DESCRIPTION
**Pull request by lipskij**
Added status icons

### CODE REVIEW

File: src/components/quad-status/BottomStatusIcons.vue
Comment (haslinghuis) [inline src/components/quad-status/BottomStatusIcons.vue:41]:
```suggestion
            return (this.auxConfig.includes("ARM") &&
                bit_check(this.mode, this.auxConfig.indexOf("ARM")));
```

File: src/components/quad-status/BottomStatusIcons.vue
Comment (haslinghuis) [inline src/components/quad-status/BottomStatusIcons.vue:53]:
```suggestion
            return (this.auxConfig.includes("FAILSAFE") &&
                bit_check(this.mode, this.auxConfig.indexOf("FAILSAFE")));
```

File: src/components/quad-status/BottomStatusIcons.vue
Comment (haslinghuis) [inline src/components/quad-status/BottomStatusIcons.vue:60]:
```suggestion
            return (performance.now() - this.lastReceivedTimestamp < 300);
```

**Review by haslinghuis — COMMENTED** [review]
Fixing some sonar complaints :)

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by chmelevskij — APPROVED** [review]

**Pull request merged**
